pytermgui/ansi_interface.py

```diff
@@ -234,11 +224,24 @@ def _call_listener(self, event: int, data: Any) -> None:
             for callback in self._listeners[event]:
                 callback(data)
 
+    async def _WinSIGWINCH(self, check_again_in: float = 0.5) -> None:
+        """Asynchronous replacement for `signal`'s *SIGWINCH*"""
+        while True:
```

Comment:
The project always has a linebreak between the docstring and body of a function.

Reply:
I formatted it to did that